src/Pages/Events/EventCard.js

```diff
@@ -16,38 +15,35 @@ function EventCard(props) {
   } = props;
 
   return (
-    <Container
-      className='event-container'
-      onClick={() => {
-        handleClick();
-      }}
-    >
-      <Row className='card-row'>
-        <Col>
+    <div>
+      <button
+        className='event-card-button'
+        onClick={() => {
+          handleClick();
+        }}
+      >
+        <tbody>
+          <td>
+            <img id='event-image'
+              className='event-image'
+              src={imageURL} alt=''
+            />
+          </td>
+          <Row className='event-title'>{title.slice(0, 24)}</Row>
           <Row className='event-date'>
-            {clockSymbol()}{'  '}
-            {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} - {endTime}
+            {clockSymbol()}
+            {'  '}
+            {getDateWithSlashes(eventDate.slice(0, 10))} {startTime} -
+            {' '}
+            {endTime}
           </Row>
-          <Row className='event-title'>{title}</Row>
           <Row className='event-location'>
             {mapPinSymbol()}
-            {eventLocation}
+            {eventLocation.slice(0, 28)}
```

Comment:
is there a reason why we only keep the first 28 characters?

if you absolutely need this, consider limiting the users input for creating an event instead of handling it later here.

Reply:
Thank you for the suggestion. I just implemented code to limit the users' input instead.